basis/http/client/client-tests.factor

```diff
@@ -27,6 +28,7 @@ IN: http.client.tests
 {
     T{ request
         { url T{ url { protocol "https" } { host "www.amazon.com" } { port 443 } { path "/index.html" } } }
+        { proxy-url T{ url } }
```

Comment:
Oops.
(But the code can be written differently. It might be more complex but bot that much i guess)